src/components/compose.jsx

```diff
@@ -600,15 +602,27 @@ function Compose({
     const handleItems = (e) => {
       const { items } = e.clipboardData || e.dataTransfer;
       const files = [];
+      const unsupportedFiles = [];
       for (let i = 0; i < items.length; i++) {
         const item = items[i];
         if (item.kind === 'file') {
           const file = item.getAsFile();
-          if (file && supportedMimeTypes.includes(file.type)) {
+          if (supportedMimeTypes !== undefined && !supportedMimeTypes.includes(file.type)) {
+            unsupportedFiles.push(file);
+          }
+          else {
```

Comment:
I think prettier doesn't seem to run on this line? Should be `} else {` without new line.